cosmos-proxy/src/logger.js

```diff
@@ -24,33 +24,37 @@
 
 // Module dependencies
 var config = require('../conf/cosmos-proxy.json');
-var winston = require('winston');
+    winston = require('winston');
 
 // Global variables
-var logFileName = config.log.file_name;
-var logDatePattern = config.log.data_pattern;
-
-// Winston configuration
-winston.add(winston.transports.DailyRotateFile, {
-    filename: logFileName,
-    datePattern: logDatePattern
-});
-
-function info(message) {
-    winston.info(message);
-} // info
-
-function error(message) {
-    winston.error(message);
-} // error
-
-function warn(message) {
-    winston.warn(message);
-} // warn
-
-function debug(message) {
-    winston.debug(message);
-} // debug
+try {
```

Comment:
Deleted `try-catch` here, as you can see in the fix I've created a new class for the check method, fixing this comment and the next one :) 
Fixed in d83de7c